packages/meditrak-app/app/country/RequestCountryAccessPage.js

```diff
@@ -194,7 +194,7 @@ export class RequestCountryAccessPage extends React.Component {
                 <Button
                   title="Request access"
                   isDisabled={isComplete}
-                  onPress={() => this.submitForm()}
+                  onPress={e => this.submitForm(e)}
```

Comment:
I don't think this is required here (the `submitForm` method does not accept any params. No functionality harm passing it in either

Reply:
The double review digs deep...thanks Kostas, have updated directly in dev